docs/changelog.txt

```diff
@@ -59,6 +59,8 @@ Template for new versions:
 
 ## Misc Improvements
 
+- `suspendmanager`: Take in account walls planned on the z-layer below to determine accessibility to a job
```

Comment:
Not sure if it is a bug fix or improvement, and the wording is hard here :)

Reply:
Clearly an improvement, there never was the expectation to handle all cases with our heuristics.